src/MicrosoftTeams.ts

```diff
@@ -2119,4 +2119,43 @@ namespace microsoftTeams {
       ]);
     }
   }
+
+  /**
+   * @private
+   * Hide from docs
+   * --------
+   * Information about all members in a chat
+   */
+  export interface ChatMembersInformation {
+    members: ChatMember[];
+  }
+
+  /**
+   * @private
+   * Hide from docs
+   * --------
+   * Information about a chat member
+   */
+  export interface ChatMember {
+    /**
+     * The user name of the user principal.
```

Comment:
Let's change this to "the member's user principal name in the current tenant"
"User name" could be confusing, because in a UPN like xxx@example.com, technically "xxx" is the "user name".

Reply:
Updated